trie/verkle.go

```diff
@@ -128,10 +128,6 @@ func (t *VerkleTrie) GetAccount(addr common.Address) (*types.StateAccount, error
 		return nil, nil
 	}
 
-	if len(values[utils.BasicDataLeafKey]) > 0 {
```

Comment:
Considering the code right above, this `if` can't be `false` (i.e `!emptyAccount`). 
Another way to see it is looking at L144. For the `balance` we didn't do this check either.